Dec

alxrwagner opened the pull request.

alxrwagner committed:
Between

alxrwagner committed:
Between

alxrwagner committed:
Between

alxrwagner committed:
Between

alxrwagner committed:
Between

alxrwagner committed:
почистил от неиспользуемых импортов

alxrwagner committed:
убрал неиспользуемые инжекты и методы

alxrwagner committed:
Добавил файл README

alxrwagner committed:
Редактировал файл README

alxrwagner committed:
Добавил метод поиска по названию

alxrwagner committed:
Эксперименты

alxrwagner committed:
Эксперименты

alxrwagner committed:
Редактировал доступ и смену пароля

alxrwagner committed:
Исправил доступ к методам

alxrwagner committed:
tests

alxrwagner committed:
tests

alxrwagner committed:
clear

alxrwagner committed:
Добавил валидатор

alxrwagner committed:
валидатор

alxrwagner committed:
валидатор

alxrwagner committed:
Документация

alxrwagner committed:
Добавил пару тестов

alxrwagner committed:
Completed

SergeyBasharkin commented on src/main/java/ru/skypro/homework/WebSecurityConfig.java:
```diff
@@ -1,60 +1,52 @@
+            "/register",
+            "/ads",
+            "/ads/image/**",
+            "/ads/search"
```
забыл дать доступ на /users/me/image/

SergeyBasharkin commented on src/main/java/ru/skypro/homework/controller/AdsController.java:
```diff
@@ -0,0 +1,155 @@
+
+
+@RestController
+@CrossOrigin(value = "http://localhost:3000")
```
Посмотри про CorsRegistry, будет полезно

SergeyBasharkin commented on src/main/java/ru/skypro/homework/service/converter/Converter.java:
```diff
@@ -0,0 +1,20 @@
+import org.springframework.stereotype.Component;
+
+@Component
+public class Converter extends AbstractJackson2HttpMessageConverter {
```
Неудачный пакет для конвертера. Это же не сервис. С остальными классами тоже самое

SergeyBasharkin commented on src/main/java/ru/skypro/homework/service/AdsService.java:
```diff
@@ -0,0 +1,103 @@
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        User user = userRepos.findByUsername(authentication.getName()).orElseThrow(NotFoundException::new);
```
в Authentication и так есть пользователь, зачем его доставать

alxrwagner committed:
- добавил глобальную конфигурацию cors
- добавил урл к картинке в белый лист
- изменил пакеты некоторых классов
- изменил метод обновления пароля и картинки пользователя

alxrwagner committed:
Исправление получения юзера через authentication, небольшая чистка кода
